mmdet/models/losses/dice_loss.py

```diff
@@ -11,10 +11,9 @@ def dice_loss(pred,
               weight=None,
               eps=1e-3,
               reduction='mean',
+              naive_dice=False,
               avg_factor=None):
-    """Calculate dice loss, which is proposed in
```

Comment:
We should keep line 16-17 in Notes and discuss that there are two main-stream implementation of dice loss.